### DISCUSSION TITLE
Support for embedding youtube videos in blog posts

### DISCUSSION THREAD

**Discussion by ashwanth1109** [LekoArts/gatsby-themes#622] [Questions] [answered]
Hi, I'm looking to add youtube videos to my blog. I tried using this [plugin](https://www.gatsbyjs.com/plugins/gatsby-remark-embed-video/) but that didnt seem to work with the theme (I'm using `gatsby-starter-minimal-blog` template).

Can we add support for this? I'm happy to propose and try to implement this feature with a little bit of guidance. Please let me know.

**Comment by ashwanth1109** [comment, marked as answer]
My mistake. I was able to integrate the video plugin by overriding mdx from the theme. Your answer from this [discussion](https://github.com/LekoArts/gatsby-themes/discussions/614) was very helpful. What threw me off a bit was disabling mdx (I didnt understand how to do it).

In case someone else is having trouble with figuring out how to set mdx, it can be done as follows:
```
{
    resolve: `@lekoarts/gatsby-theme-minimal-blog`,
    // See the theme's README for all available options
    options: {
        mdx: false,
        externalLinks: [...],
    },
},
```

**Reply by LekoArts** [reply]
Glad you were able to solve this!
